README.md

```diff
@@ -0,0 +1,116 @@
+# IonicSDK for Ionic 5.x/ Vue applications
+
+## How to use?
+
+- Clone your repository  
+`git clone <your repository url>`
+
+- Create a new branch
+- Add IonicSDK repo URL to remote  
```

Comment:
Replace ionicSDK with Order Routing Rules

Reply:
Made the requested change